spec/xcpretty/parser_spec.rb

```diff
@@ -180,6 +180,13 @@ module XCPretty
       @parser.parse(SAMPLE_PENDING_KIWI_TEST)
     end
 
+    it "parses measuring tests" do
+      @formatter.should receive(:format_measuring_test).with('SecEncodeTransformTests.SecEncodeTransformTests',
+                                                           'test_RFC4648_Decode_UsingBase32',
```

Comment:
Line is too long. [93/80]<br>Align the parameters of a method call if they span more than one line.